### ISSUE TITLE
IR Remote without function

### ISSUE THREAD

**Issue by meikelmoeller** [codesnake/OpenELEC.tv#63]
"OpenELEC-Amlogic.M8.arm-5.0.3" runs on my MXIII. Everything works fine but the remote control is out of function. Is there a possibility getting the remote control working? Without remote control, the box is nearly useless for me.

**Comment by kszaq** [comment]
Simply grab a _remote.conf_ file from Android firmware for your box and put it in _Config_ Windows share. After reboot it should work. Alternatively, you can try different _remote.conf_ files for other devices. Look for them in_device_/filesystem/etc folders in the repo.

**Comment by huggie007** [comment]
Does that require taking box to android firmware then taking remote.config file and then putting openelec on to box then dropping remote.config file in there.

**Comment by kszaq** [comment]
No, you can grab the file from the update zip. If you don't have one, you can try using configuration files from other boxes - they are available in @codesnake's repository as I mentioned before. There are not many remotes used for Amlogic devices and it's highly possible one of them will work.

**Comment by huggie007** [comment]
I'm sure I have the correct remote.config file that I require. But now the
problem I'm having is transferring it to box. I'm using file zila under
port 22 which is correct for ssh. However when I try to overight it in etc
folder. It asks me to overight which I do then fails and says could not
start transfer. Where am I going wrong. Help needed please. Thanks in
advance

**Comment by codesnake** [comment]
@huggie007 You need to put your remote.conf into /storage/.config folder.

**Comment by meikelmoeller** [comment]
Where can I find the right remote.conf for the MXIII 2GB. I have no vendor, only  MXIII.

**Comment by kszaq** [comment]
@meikelmoeller try this one: https://raw.githubusercontent.com/kszaq/OpenELEC.tv/openelec-5.0/projects/Amlogic/devices/MK808B.Plus/filesystem/etc/remote.conf

**Comment by meikelmoeller** [comment]
Yeah, it works, great job and thx for the support.

**Comment by huggie007** [comment]
i grabbed the remote from link above and tried it on m8s which if im not
mistaken is the same remote as mxiii. but some work and some dont. have i
got correct file or do i need to remap all keys.

**Comment by huggie007** [comment]
im still stuck trying to get remote to work correctly with m8s box. i get
some buttons to work and some dont work at all. is there a list available
with codes to correct buttons.

**Comment by BOFH2k** [comment]
Configuring my remote for the S82 round box works fine in 6.0.1 manually:
remotecfg /storage/.config/remote.conf
But that config is not applied upon reboot...
